### ISSUE TITLE
[RFC] Upgrade HTML output

### ISSUE THREAD

**Issue by AyanSinhaMahapatra** [nexB/scancode-toolkit#3281]
## Short Description

We should also include and have support for the following CLI options:

- license text
- license text diagnostics
- license-clarity-score
- summary (?)
- license references (?)

and others options also potentially, which are default in JSON/YAML.

## Possible Labels

- new feature

## Select Category

-  [x] Enhancement
-  [ ] Add License/Copyright
-  [ ] Scan Feature
-  [ ] Packaging
-  [ ] Documentation
-  [x] Expand Support
-  [ ] Other

**Comment by Inzamamdev** [comment]
can I work on this issue

**Comment by AyanSinhaMahapatra** [comment]
@456535688765 yes! You do not need to ask for permission. Though please select one issue at a time instead of selecting a bunch :P

**Comment by BRGOVIND** [comment]
HELLO
since there was no PR from the previous guy
can i do this ?
